packages/ui/src/composites/wui-qr-code/styles.ts

```diff
@@ -0,0 +1,51 @@
+import { css } from 'lit'
+
+export default css`
+  :host {
+    position: relative;
+    user-select: none;
+    display: block;
+    overflow: hidden;
+    aspect-ratio: 1 / 1;
+    animation: fadeIn ease 0.2s;
```

Comment:
In v3, we won't need fade in animation as qrcode will be available upon opening the modal